supervisor/pidproxy.py

```diff
@@ -7,22 +7,27 @@
 import sys
 import signal
 import time
+import argparse
```

Comment:
I haven't really reviewed the rest of this pull request but Supervisor works on Python 2.6+ so it can't use `argparse`.